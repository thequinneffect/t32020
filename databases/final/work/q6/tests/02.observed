tests/02.sh: 1: ./q6: Permission denied
