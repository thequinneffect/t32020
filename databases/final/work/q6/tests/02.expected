Discography for Roaring Gang
--------------------
Waiting for Home (2010) (rock)
--------------------
 1. Dreams of Defeat (5:01)
 2. Fast Train to Tokyo (9:40)
 3. Kill for Summer (1:11)
 4. Feel Life (5:14)
 5. Take the Clouds (6:03)
 6. Up to Haze (1:55)
 7. Fast Train to the Heat (4:48)
 8. Fly to this World (4:37)
 9. Beyond White (5:40)
10. Feel Khatmandu (1:28)
--------------------
Up to Triumph (2011) (rock)
--------------------
 1. Escape from the Horizon (3:29)
 2. Pump up Haze (3:20)
 3. Master of this World (8:13)
 4. Maybe Pandemic (3:43)
 5. Kill for Fire (7:22)
 6. Over Solace (1:27)
 7. Why the Torpedoes (4:50)
 8. Why not Khatmandu (4:44)
 9. What happened to New York (3:56)
10. What about the Moon (4:55)
11. Touch the Torpedoes (1:13)
--------------------
What happened to Night (2012) (hip-hop)
--------------------
 1. Beyond the Dance Floor (6:13)
 2. Get out of Plague (11:06)
 3. Run with the Heat (11:48)
 4. Ode to this City (3:55)
 5. Master of Haze (5:13)
 6. Bring the Ocean (10:13)
 7. Where is Dancing (12:19)
 8. Why Destruction (6:20)
 9. Where is Fear (3:52)
